src/Components/Subscriptions/Table/index.jsx

```diff
@@ -71,23 +76,25 @@ const Table = ({ data }) => {
   };
 
   const closeModal = () => {
+    setShowConfirmDeleteModal(false);
     setShowDeleteModal(false);
   };
 
   return (
-    <div>
-      <table className={styles.tableContainer}>
-        <thead>
+    <div className={styles.container}>
+      <table>
+        <thead className={styles.containerThead}>
           <tr>
-            <th>Classes</th>
-            <th>Members</th>
-            <th>Date</th>
+            <th className={styles.thead}>Classes</th>
+            <th className={styles.thead}>Members</th>
```

Comment:
Hi! Here its not necesary to add a className to each <th> tag, you can directly use the <thead> to apply x style and it will automaticaly apply to every <th> inside